validations.py

```diff
@@ -0,0 +1,181 @@
+import argparse
```

Comment:
I'm not sure the best place for this, but lets creating a directory for this little project outside of the root.  Maybe somewhere in the ops directory, similar to the load testing package (I'm open to ideas).

In the project directory, lets create a README with detailed instructions on how to get ones local environment setup to use this, we should probably include a requirements.txt file there for installing yaml (or we could just convert the confg to TOML or JSON, both of which have libraries builtin to python std lib for parsing).